### ISSUE TITLE
Block failed verification during bootstrapping because "prohibited address cannot be called"

### ISSUE THREAD

**Issue by xyrus02** [ava-labs/avalanchego#1158]
**Describe the bug**
Our validators on Fuji are unable to launch successfully because of what it seems as a bad block on the C-Chain. We receive the error message:

```
ERROR <C Chain> bootstrap/block_job.go:88 block failed verification during bootstrapping {"blkID": "jAq2RCRJjzmcHqD3kX3oyZn87JzVKsu1n7VyGatmF9hvqePH5", "error": "could not apply tx 0 [0xc36fe2c7e277e5faa349816d6f5917c49c0b14e4105d17252f37b05684265ce4]: prohibited address cannot be called"}
```

**To Reproduce**
We have no information on how we can reproduce this issue but we are open to support the investigation with information from our nodes.

**Expected behavior**
A clear instruction on how to resume healthy node operation is expected as the result of the ticket.

**Screenshots**
We do not have screenshots but the error message is shown above.

**Logs**
See above

**Metrics**
We do not have any metrics.

**Operating System**
Ubuntu 22.04

**Additional context**

Avalanche Bug Bounty program can be found [here](https://hackenproof.com/avalanche/avalanche-protocol).

**Comment by bafu** [comment]
I also failed to bootstrap C-Chain with the environment

1. avalanchego v1.9.0-fuji
2. subnet-evm: v0.3.0
3. network: Fuji

Error logs

```
$ ./avalanchego \
    --whitelisted-subnets=${SUBNET_ID} \
    --network-id=fuji \
    --http-host=0.0.0.0
...
[10-23|11:52:55.029] INFO <C Chain> bootstrap/bootstrapper.go:469 fetching blocks {"numFetchedBlocks": 590000, "numTotalBlocks": 592303, "eta": "17s"}
[10-23|11:53:30.930] INFO <C Chain> bootstrap/bootstrapper.go:528 executing blocks {"numPendingJobs": 592303}
[10-23|11:53:30.944] ERROR <C Chain> bootstrap/block_job.go:88 block failed verification during bootstrapping {"blkID": "jAq2RCRJjzmcHqD3kX3oyZn87JzVKsu1n7VyGatmF9hvqePH5", "error": "could not apply tx 0 [0xc36fe2c7e277e5faa349816d6f5917c49c0b14e4105d17252f37b05684265ce4]: prohibited address cannot be called"}
[10-23|11:53:30.962] FATAL <C Chain> handler/handler.go:283 shutting down chain {"reason": "received an unexpected error", "error": "failed to execute job jAq2RCRJjzmcHqD3kX3oyZn87JzVKsu1n7VyGatmF9hvqePH5 due to failed to verify block in bootstrapping: could not apply tx 0 [0xc36fe2c7e277e5faa349816d6f5917c49c0b14e4105d17252f37b05684265ce4]: prohibited address cannot be called while processing sync message: ancestors:{chain_id:\"\\x7f\\xc9=\\x85\\xc6\\xd6,...
[10-23|11:53:31.078] INFO <C Chain> bootstrap/bootstrapper.go:282 shutting down bootstrapper
INFO [10-23|11:53:31.079] <C Chain> github.com/ava-labs/coreth/core/tx_pool.go:457: Transaction pool stopped 
INFO [10-23|11:53:31.080] <C Chain> github.com/ava-labs/coreth/core/blockchain.go:703: Stopping Acceptor 
INFO [10-23|11:53:31.081] <C Chain> github.com/ava-labs/coreth/core/blockchain.go:706: Acceptor queue drained t="51.888µs"
INFO [10-23|11:53:31.081] <C Chain> github.com/ava-labs/coreth/core/blockchain.go:708: Shutting down state manager 
INFO [10-23|11:53:31.081] <C Chain> github.com/ava-labs/coreth/core/blockchain.go:713: State manager shut down t="2.215µs"
INFO [10-23|11:53:31.081] <C Chain> github.com/ava-labs/coreth/core/blockchain.go:720: Shutting down sender cacher 
INFO [10-23|11:53:31.081] <C Chain> github.com/ava-labs/coreth/core/blockchain.go:724: Closing scope 
INFO [10-23|11:53:31.081] <C Chain> github.com/ava-labs/coreth/core/blockchain.go:727: Blockchain stopped
```

Happy to share the avalanchego logs if it will be helpful.

**Comment by xyrus02** [comment]
Hello, @bafu 

As I have learned from Avalabs team on another channel, this has to do with a security issue mitigation during 1.8.6 which, in our case, we have not installed prior to installing 1.9.0. This means the block containing the offending address got validated by our nodes, which then corrupted the state database.

The solution for us was to resync our nodes. As we do not need historical data, using state sync in the C-chain configuration was good enough for us and luckily, the whole operation took us only 2-3 hours.

Hoping to read an official response from Avalabs team on this item.

**Comment by bafu** [comment]
@xyrus02 Thanks for sharing the helpful information!

I've fixed this issue as well, and here is my solution:

1. In my case, the updating path of my nodes was 1.7.x -> 1.8.2 -> 1.8.4 -> 1.8.6 -> 1.9.0.
    1. My nodes had experienced the security issue migration, and that's different from Xyrus' case a bit.
2. I found that my nodes were two patches behind the latest 1.9.0: [1.9.0-fuji-post-upgrade](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0-fuji-post-upgrade) & [1.9.0-fuji-startup](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0-fuji-startup).
3. I updated my nodes from 1.9.0 to 1.9.0-fuji-startup directly. We can skip 1.9.0-fuji-post-upgrade.

Hoping that helps who are encountering the same issue.
